services/can_logger/can_logger.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import socket
+import postgres
+import csv
+import os
+
+def write_to_csv(timestamp, can_id, can_data, host_interface):
+
+
+    with open('/data/log/' + host_interface + '.csv', mode = 'a') as log:
+
+        log = csv.writer(log, delimiter = ',', quotechar = '"',
+                         quoting = csv.QUOTE_MINIMAL)
+
+        log.writerow([timestamp, can_id, can_data])
+
+def write_to_db(timestamp, can_id, can_data):
+
+    db.run("INSERT INTO cell (time, can_id, can_data) VALUES (to_timestamp(%s), \
+            %s, %s)", (timestamp, can_id, can_data))
+
+    print("Finished inserting CAN bus data for timestamp ", timestamp)
```

Comment:
Should either remove this or also put a print statement in `write_to_csv()`. Prob want to remove, a print per CAN msg will clog up stdout very quickly